File: VB/MyPaintStyle.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraBars.Styles
Imports DevExpress.XtraBars.ViewInfo
Imports System.Drawing
Imports DevExpress.XtraBars.Docking
Imports DevExpress.Skins
Imports DevExpress.XtraBars.Docking.Paint
Imports DevExpress.Utils.Drawing
Imports System.Drawing.Drawing2D
Imports DevExpress.XtraBars.Docking.Helpers

Namespace WindowsFormsApplication1
    Public Class MyPaintStyle
        Inherits SkinBarManagerPaintStyle

        Public Sub New(ByVal collection As BarManagerPaintStyleCollection)
            MyBase.New(collection)
        End Sub
        Public Overrides ReadOnly Property Name() As String
            Get
                Return "MyPaintStyle"
            End Get
        End Property
        Protected Overrides Sub CreatePainters()
            MyBase.CreatePainters()
            Me.fElementsPainter = New MySkinElementPainter(Me)
        End Sub
    End Class

    Public Class MySkinElementPainter
        Inherits DockElementsSkinPainter

        Public Sub New(ByVal paintStyle As SkinBarManagerPaintStyle)
            MyBase.New(paintStyle)
        End Sub

        Protected Overrides Sub DrawActiveTab(ByVal tabArgs As DrawTabArgs)

            tabArgs.Appearance.BackColor = Color.Chocolate
            tabArgs.Appearance.BackColor2 = Color.Cornsilk
            tabArgs.Appearance.GradientMode = LinearGradientMode.Vertical
            tabArgs.DrawBackground()
            Dim rect As New DirectionRectangle(PaintHelper.CalcTabContentBounds(tabArgs, DockingSkins.SkinTabHeader), Not tabArgs.IsVertical)
            rect.CutFromHead(TabPanelPainter.TabHorzTextIndent)
            rect.CutFromTail(1)
            tabArgs.DrawString(tabArgs.TabLayout.TabText, rect.Bounds)
        End Sub
    End Class
End Namespace

File: CS/MyPaintStyle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraBars.Styles;
using DevExpress.XtraBars.ViewInfo;
using System.Drawing;
using DevExpress.XtraBars.Docking;
using DevExpress.Skins;
using DevExpress.XtraBars.Docking.Paint;
using DevExpress.Utils.Drawing;
using System.Drawing.Drawing2D;
using DevExpress.XtraBars.Docking.Helpers;

namespace WindowsFormsApplication1
{
    public class MyPaintStyle : SkinBarManagerPaintStyle
    {
        public MyPaintStyle (BarManagerPaintStyleCollection collection) : base(collection) { }
        public override string Name { get { return "MyPaintStyle"; } }
        protected override void CreatePainters()
        {
            base.CreatePainters();
            this.fElementsPainter = new MySkinElementPainter(this);
        }        
    }

    public class MySkinElementPainter : DockElementsSkinPainter
    {
        public MySkinElementPainter(SkinBarManagerPaintStyle paintStyle) : base(paintStyle) { }

        protected override void DrawActiveTab(DrawTabArgs tabArgs)
        {

            tabArgs.Appearance.BackColor = Color.Chocolate;
            tabArgs.Appearance.BackColor2 = Color.Cornsilk;
            tabArgs.Appearance.GradientMode = LinearGradientMode.Vertical;
            tabArgs.DrawBackground();
            DirectionRectangle rect = new DirectionRectangle(PaintHelper.CalcTabContentBounds(tabArgs, DockingSkins.SkinTabHeader), !tabArgs.IsVertical);
            rect.CutFromHead(TabPanelPainter.TabHorzTextIndent);
            rect.CutFromTail(1);
            tabArgs.DrawString(tabArgs.TabLayout.TabText, rect.Bounds);
        }    
    }
}
